### PR TITLE
[NPU] Support npu kernel for op elementwise_floordiv

### PR DESCRIPTION
**Pull request by liym27**
### PR types

New features
### PR changes

OPs
### Describe

Support npu kernel for op elementwise_floordiv

image

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/npu/test_elementwise_floordiv_op_npu.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        }
+        self.attrs = {}
+        self.outputs = {'Out': self.out}
+
+    def set_npu(self):
+        self.__class__.use_npu = True
+        self.place = paddle.NPUPlace(0)
+
+    def init_input_output(self):
+        self.x = np.random.uniform(1, 2, [10, 10]).astype(self.dtype)
```
Comment (pangyoki) [inline python/paddle/fluid/tests/unittests/npu/test_elementwise_floordiv_op_npu.py]: 测试数据里x和y全为1吗？需不需要测多几种情况？例如 5 / 2 ？

Comment (liym27) [reply python/paddle/fluid/tests/unittests/npu/test_elementwise_floordiv_op_npu.py]: 好的，已测试~

**Review by pangyoki — APPROVED** [review]
LGTM

**Review by zhiqiu — APPROVED** [review]
LGTM

**Pull request merged**